composer.json

```diff
@@ -46,7 +46,7 @@
         "psr/http-client": "^1.0",
         "psr/http-message": "^1.0",
         "psr/log": "^1.1",
-        "shopware/core": "^6.4.11",
+        "shopware/core": "~v6.4.0",
```

Comment:
Afaik this is on purpose. Shopware broke some backwards compatibility in their administration javascript with this version, so we only support versions from this point on. But I'll have to check with @JoshuaBehrens to make sure, I remember this correctly.